src/strings/index.tsx

```diff
@@ -664,6 +664,30 @@ const strings = new LocalizedStrings({
     SET_UP: 'Set Up',
     SUBSET_ERROR: 'Subset weight should be less than seed weight',
     WITHDRAWN_QUANTITY_ERROR: 'Exceeds remaining quantity',
+    TOOLTIP_SPECIES_TABLE_SCIENTIFIC_NAME:
+      'The binomial Latin name (genus + species) currently accepted in the botanical literature.',
+    TOOLTIP_SPECIES_TABLE_COMMON_NAME:
+      'The name an organism is known by to the general public, rather than its scientific name.',
+    TOOLTIP_SPECIES_TABLE_FAMILY:
+      'The scientific name of the plant family currently accepted in the botanical literature.',
+    TOOLTIP_SPECIES_TABLE_GROWTH_FORM:
+      'A structural category consisting of individuals or species of the same general habit of growth but not necessarily related. Learn more.',
```

Comment:
Does it make sense to put "Learn more." into its own string, like a `TOOLTIP_LEARN_MORE`. That way you can pass it into the tooltip component separately to allow it to be rendered differently (with its link/dialog trigger)?

Reply:
@charlie-tf it absolutely makes sense and I intend to do so before marking this PR as ready for review

Reply:
I included the "Learn more." in the strings as placeholders for now to help me remember which tooltips will need to be updated 🎗️